R/plot-residuals-vs-simulated.R

```diff
@@ -43,20 +44,25 @@
 #' myPlotConfiguration$caption <- "My Sources"
 #'
 #' # plot
-#' plotResidualsVsSimulated(myDataCombined, myPlotConfiguration)
+#' plotResidualsVsSimulated(myDataCombined, scaling = "log", defaultPlotConfiguration = myPlotConfiguration)
 #'
 #' @export
 plotResidualsVsSimulated <- function(dataCombined,
-                                     defaultPlotConfiguration = NULL) {
+                                     defaultPlotConfiguration = NULL,
```

Comment:
why is it actually called **default**PlotConfiguration and not just plotConfiguration?

Reply:
Because `PlotConfiguration` is already used in `TLF`...

Reply:
?? it's a function argument, you can call it whatever you want.
```

plotResidualsVsSimulated <- function(dataCombined,
                                     plotConfiguration = NULL,
                                     scaling="lin") {
```

Not a big deal, we can keep it AS IS. Just wondering.

Reply:
Ah no I understand, you mean the argument name. I prefer to have the class name so it is clear why type is required.